apps/api/src/.env.agent

```
@@ -0,0 +1,104 @@
+
+# expressed in seconds or a string describing a time span [zeit/ms](https://github.com/zeit/ms.js).  Eg: 60, "2 days", "10h", "7d"
+SUBSCRIBER_WIDGET_JWT_EXPIRATION_TIME='15 days'
+
+
+# Cloudflare Scheduler (for delayed job scheduling)
+SCHEDULER_URL=
+SCHEDULER_API_KEY=
+SCHEDULER_CALLBACK_API_KEY=
+
+NOVU_ENTERPRISE=true
+BETTER_AUTH_SECRET=0753ac1a608b4534fa69297c386e3d74024fed295459ae0ea9fcb01cc555190b
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The BETTER_AUTH_SECRET key should go before the NOVU_ENTERPRISE key

(UnorderedKey)